torch/onnx/__init__.py

```diff
@@ -239,8 +278,27 @@ def _run_symbolic_function(g, n, inputs):
         raise
 
 
+# Generate an ONNX ATen op node. The generated op keeps ATen attribute names.
 def _graph_at(g, opname, *args, **kwargs):
-    return g.op("ATen", *args, operator_s=opname, **kwargs)
+    # If aten is True, attribute names should not contain the type suffix.
+    aten = kwargs.pop("aten", False)
+
+    # Handle the multiple return ATen ops.
+    # If ATen ops have more than one return values, set the outputs.
+    # The exporter generates select nodes.
+    outputs = 1
+    if opname == "max_pool2d":
```

Comment:
You don't want to hard code multiple return ATen operators here; if we add more multiple return ops (and we will), these lists will go out of sync, and it's an easy way to end up with bugs. And furthermore, we already have a canonical source of information for the actual number of outputs: the outputs of the node we are translating! You can just use `len(node.outputs())` to figure out the number of outputs (and if you follow my advice in the other comment, you'll have access to `node`, even though you don't in this function).

Reply:
I totally agree with you. I also hate hard code things.